apps/prairielearn/src/tests/sprocs/users_select_or_insert.test.js

```diff
@@ -32,6 +32,18 @@ async function usersSelectOrInsert(user, authn_provider_name = null, institution
   ]);
 }
 
+/*
```

Comment:
I wanted to keep it, even if unused, but the typechecking didn't like it. We could either remove it because it's small, leave it commented out, or otherwise force the checker not to care that nothing in the live test is running it.

Reply:
If you really want to save this somewhere, stick it in some file like `tests/utils/audit-logs.ts` and export it. Otherwise, this is the kind of debugging code that I'd tend to just write on the fly when I need it; it's not complex enough to warrant keeping it around in a comment.

Reply:
Resolved with b65159bef